Texstyle/Texstyle/ControlStateText.swift

```diff
@@ -0,0 +1,177 @@
+//
+// Copyright (c) 2019 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+/// Stores string, styles and substyles for attributed string depending on COntrolState.
+public final class ControlStateText {
+    public let value: String
+    public let styles: [ControlState: TextStyle]
+
+    /// Returns attributed string for normal state.
+    public var attributed: NSAttributedString? {
+       attributed()
+    }
+
+    /// Clears cached attributed string.
+    var substyles: [ControlState: [TextSubstyle]] = [:] {
```

Comment:
I know that's my docs, but please add an additional comment what are substyles actually.